### PR TITLE
align relay log behaviour in v5.4.0

### PR DESCRIPTION
**Pull request by TomShawn**
### What is changed, added or deleted? (Required)
align relay log behaviour in v5.4.0

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [x] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-cn/pull/8165 https://github.com/pingcap/docs-cn/pull/8113 https://github.com/pingcap/docs-cn/pull/8107
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by hfxsd — APPROVED** [review]
Rest LGTM

**Review by lance6716 — COMMENTED** [review]
rest lgtm

File: dm/dm-worker-configuration-file.md [hunk 1]
```diff
@@ -27,6 +27,7 @@ join = "http://127.0.0.1:8261,http://127.0.0.1:8361,http://127.0.0.1:8461"
 
 keepalive-ttl = 60
 relay-keepalive-ttl = 1800 # New in DM v2.0.2.
+# relay-dir = "relay_log" # New in 5.4.0. When you use a relative path, determine the path according to the deployment and start method.
```
[the remark this answers is not in the document]
Comment (TomShawn) [reply dm/dm-worker-configuration-file.md:30]:
```suggestion
# relay-dir = "relay_log" # New in 5.4.0. When you use a relative path, check the deployment and start method of DM-worker to determine the full path.
```

[the remark this answers is not in the document]
Comment (TomShawn) [reply dm/dm-worker-configuration-file.md]: Done

**Review by lance6716 — APPROVED** [review]

**Pull request merged**